index.js

```diff
@@ -0,0 +1,157 @@
+const HTML_CN = {
+	disable: 'disable',
+};
+
+const selector = {
+	start: document.querySelector('.start'),
+	guess: document.querySelector('.guess'),
+    input: document.querySelector('#input'),
```

Comment:
해당 부분만 인덴팅이 어긋났습니다.
selector를 객체화 한 부분도 아주 좋습니다 👍
```suggestion
      input: document.querySelector('#input'),
```